hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -796,7 +796,7 @@ static void deltaStreamerTestRunner(HoodieDeltaStreamer ds, HoodieDeltaStreamer.
         throw new RuntimeException(ex.getMessage(), ex);
       }
     });
-    TestHelpers.waitTillCondition(condition, dsFuture, 360);
+    TestHelpers.waitTillCondition(condition, dsFuture, 720);
```

Comment:
Which test case does it fix ?

Reply:
TestHoodieDeltaStreamerWithMultiWriter.testUpsertsContinuousModeWithMultipleWritersWithoutConflicts